### ISSUE TITLE
FusedBatchNorm Conversion gives 5 Outputs

### ISSUE THREAD

**Issue by ram1897** [onnx/tensorflow-onnx#143]
Hi, when the Model containing Fused Batch Norm operation is converted, there are 5 outputs generated. Is there any flag or any way to **enable only a single output to be produced** for this case.

**Comment by pengwa** [comment]
@ram1897  I recalled in training mode, Fused Batch Norm will have more outputs, right? this converter would only expect the ops in inference mode.

**Comment by pengwa** [comment]
this is defined by TF https://www.tensorflow.org/api_docs/cc/class/tensorflow/ops/fused-batch-norm. why not enable only a single output?

**Comment by ram1897** [comment]
What if we need the output of only the BatchNorm and not other information present in the Nodes.

**Comment by guschmue** [comment]
Functionality wise it is fine to not consume all outputs. 

But something is fishy with the docs:
In opset 7 there is no test_mode attribute but outputs 1-5 are optional:
https://github.com/onnx/onnx/blob/master/docs/Operators.md#BatchNormalization
but in opset 6 there was a test_mode:
https://github.com/onnx/onnx/blob/master/docs/Changelog.md#BatchNormalization-6

Let me ask some people, possible that there are no optional outputs anymore.

**Comment by guschmue** [comment]
Talked to @linkerzhang : since opset 7 BatchNormalization is implicitly in test_mode, aka it should produce only 1 output. The optional outputs are still in the doc in case onnx gets a training feature.
So I belief this should be doing already what you are asking for.
